src/allotropy/allotrope/schema_mappers/adm/plate_reader/benchling/_2023/_09/plate_reader.py

```diff
@@ -283,9 +272,9 @@ def _get_technique_document(
 
     def _get_measurement_document(
         self, measurement: Measurement
-    ) -> MeasurementDocumentItems:
+    ) -> MeasurementDocument:
         # TODO(switch-statement): use switch statement once Benchling can use 3.10 syntax
```

Comment:
I think this can happen?

Reply:
It's all weird auto-generated code stuff from the datamodel-codegen package. It sometimes adds a suffix and sometimes doesn't. Here changing the structure a bit made it change it's mind 🤷